sl_patches/spdk_export_set_thread.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/include/spdk/thread.h b/include/spdk/thread.h
```

Comment:
So this is the patch that you will apply in the RPM build also then.

Reply:
Yes, I'm not quite sure how to make this change in the daos spdk repository, could you help to submit a patch for this change? Thanks in advance.